src/hooks/useArrowKeyFocusManager.ts

```diff
@@ -84,7 +86,13 @@ export default function useArrowKeyFocusManager({
             while (disabledIndexes.includes(newFocusedIndex)) {
                 newFocusedIndex -= allowHorizontalArrowKeys ? itemsPerRow : 1;
                 if (newFocusedIndex < 0) {
-                    break;
+                    if (disableCyclicTraversal) {
+                        if (!allowNegativeIndexes) {
```

Comment:
Which `if` ??

Reply:
it is the exceptional case for emoji picker menu so if you remove it, it will break the emoji picker search input auto focusing.

Reply:
If we use the `break` mechanism everything will work fine but principaly that's wrong and it will create the same issue for the future. To demonstrate that remove this [block](https://github.com/Expensify/App/blob/535cad796c4bd2105c53069d24a92a0e59111892/src/hooks/useArrowKeyFocusManager.ts#L90-L92) then pass disableCyclicTraversal as true for context menu here
https://github.com/Expensify/App/blob/6e42e55c2554c1dd5ab17e274f56ab5c4069eaf1/src/pages/home/report/ContextMenu/BaseReportActionContextMenu.tsx#L157-L158
```
        disableCyclicTraversal: true,

```
Now, while you are on the first active index if you press up the focus will be lost and unless u press down it will not appear that is what it will cause if we use the break logic but now as it is the emoji picker the only exceptional case we allow negative indexes to auto focus search input we are treating it as an exception.